frontend_tests/node_tests/filter.js

```diff
@@ -715,6 +717,50 @@ run_test('predicate_basics', () => {
         display_recipient: [{id: steve.user_id}, {id: me.user_id}],
     }));
     assert(!predicate({type: 'stream'}));
+
+    const img_msg = {
+        content: "<p><a href=\"\/user_uploads\/randompath\/test.jpeg\">test.jpeg<\/a><\/p>\n<div class=\"message_inline_image\"><a href=\"\/user_uploads\/randompath\/test.jpeg\" title=\"test.jpeg\"><img src=\"\/user_uploads\/randompath\/test.jpeg\"><\/a><\/div>",
```

Comment:
We could use strings contained within single quotes or backticks to avoid the escaping here... unless it was actually simpler to use these from our `markdown_test_cases.json` directly.

Reply:
I think back-tick approach is better.